src/components/flag/flag-deprecated.jsx

```diff
@@ -5,43 +5,43 @@ import injectSheet from 'react-jss';
 import cn from 'classnames';
 import flags from './flags';
 import CurrencyFlag from './flags/currencies';
-import omit from '../../utilities/omit';
 
-const addBorder = props => !props.hideBorder && !props.round && !props.secondaryCountryCode;
+export const styles = () => ({
+  container: {
+    display: 'inline-block',
+  },
+  flagStyle: {
+    display: 'block',
+    boxSizing: 'border-box',
+    left: 0,
```

Comment:
Is this needed? It feels like this together with `position: relative` will move the flag including the border `0px` to the left.